components/rhs_comment/rhs_comment.jsx

```diff
@@ -360,7 +361,7 @@ export default class RhsComment extends React.Component {
                     />
                     <ChannelPermissionGate
                         channelId={post.channel_id}
-                        teamId={this.props.teamId}
+                        teamId={this.props.teamId || this.props.currentTeamId}
```

Comment:
Instead of embedding this logic in the view, should the connector examine `ownProps` and override `teamId` with `currentTeamId` if not set? As a client of this component, it would not be immediately obvious to me which of `teamId` or `currentTeamId` to supply.

Reply:
It makes sense, I'll do the change :)